allauth/account/views.py

```diff
@@ -290,6 +292,19 @@ def get_context_data(self, **kwargs):
         )
         return ret
 
+    def get_initial(self):
+        intial = super(SignupView, self).get_initial()
```

Comment:
Typ-o: in-i-tial

Reply:
I think the spelling is correct, it's in-i-tial

Reply:
But it says `intial = ...`